redirects.json

```diff
@@ -1481,21 +1481,9 @@
     "status":"301",
     "condition":null
   },
-  {
-    "source":"/contribute/",
-    "target":"/javascript/contribute/",
-    "status":"301",
-    "condition":null
-  },
-  {
-    "source":"/contribute/getting-started/",
-    "target":"/javascript/contribute/getting-started/",
-    "status":"301",
-    "condition":null
-  },
```

Comment:
the contribute site is at [/contribute](https://next-release-main.d1ywzrxfkb9wgg.amplifyapp.com/contribute/)